lib/cog/relay/relays.ex

```diff
@@ -109,19 +109,17 @@ defmodule Cog.Relay.Relays do
   end
 
   defp process_introduction(%{"relay" => id,
-                              "public_key" => pub_key,
                               "reply_to" => reply_to}, state) do
     case CredentialManager.get(id, by: :id) do
       {:ok, nil} ->
-        creds = %Credentials{id: id, public: Base.decode16!(pub_key, case: :lower)}
+        creds = %Credentials{id: id}
         CredentialManager.store(creds)
-        Logger.info("Stored public key for Relay #{id}")
+        Logger.info("Stored credentials for Relay #{id}")
```

Comment:
Perhaps something for a future PR, but the `Credential*` modules and wording seem out-of-place now. We should think of a better name that's more reflective of their current purpose.